homeassistant/components/mqtt/config_flow.py

```diff
@@ -137,9 +137,6 @@ async def async_step_hassio_confirm(self, user_input=None):
         return self.async_show_form(
             step_id="hassio_confirm",
             description_placeholders={"addon": self._hassio_discovery["addon"]},
-            data_schema=vol.Schema(
-                {vol.Optional(CONF_DISCOVERY, default=DEFAULT_DISCOVERY): bool}
-            ),
```

Comment:
Is this the correct change? We still want the hassio_confirm step, but without the `discovery` checkbox?

Reply:
Yes.